src/utils/actions.js

```diff
@@ -1,8 +1,13 @@
-export function getValueList(values) {
-  return values
-    .split(';')
-    .map(value => (isNaN(value) ? { qText: value } : { qIsNumeric: true, qNumber: Number(value) }));
-}
+export const getValueList = async (app, values, isDate) => {
+  const valuesArray = values.split(';');
+  if (isDate) {
+    for (let i = 0; i < valuesArray.length; i++) {
+      // eslint-disable-next-line no-await-in-loop
+      valuesArray[i] = await app.evaluate(`Num('${valuesArray[i]}')`);
```

Comment:
this is potentially very slow, but you could try and construct a single call instead?